tests/testthat/test_aorsf_surv_aorsf.R

```diff
@@ -11,3 +11,18 @@ test_that("autotest", {
     check_replicable = FALSE)
   expect_true(result, info = result$error)
 })
+
+test_that("predicted time matches aorsf", {
+  # time prediction is handled outside of aorsf.
+  # this test is placed to ensure that we don't diverge.
+  task = tsk("lung")
+  aorsf = lrn("surv.aorsf",
+              control_type = "fast",
+              na_action = "impute_meanmode")
+  aorsf$train(task)
+  preds_mlr3 = aorsf$predict(task)
+  preds_aorsf = predict(aorsf$model,
+                        new_data = task$data(),
+                        pred_type = "time")
+  expect_equal(preds_mlr3$response, as.numeric(preds_aorsf))
```

Comment:
This test fails locally (very strange) - @bcjaeger can you confirm it works, ie does the `predict` get the survival matrix only with the unique event times of the `new_data` and does the same calculation as is done manually here?
![image](https://github.com/mlr-org/mlr3extralearners/assets/10909555/7f331ddd-e9ed-4707-bedd-d4823b3d1b72)

Reply:
@bcjaeger have you had time to check this locally? Just want to verify it's just me that has the problem, the CI passes

Reply:
@bblodfon, thanks for reminding me! I will try to check this out today, but at the moment I get an error from `p_uty` regarding the `special_vals` input. I think this can be fixed if we say `default = NULL` in `p_uty`. Do you want me to try that out or would you prefer to do it?

Reply:
which version of `paradox` do you have? Strange, also happens in the CI, but works on my local

Reply:
It works with `paradox` > 1.0.0, there was a recent update and we discussed that's how it should be used, right @sebffischer ? Sadly we can't force the `paradox` version in the `DESCRIPTION` yet, since it's not on CRAN yet.

Reply:
No problem. I installed `paradox` 1.0.0 and the `aorsf` tests pass locally for me. 😕

Reply:
Could you please send your session info?

Reply:
Definitely.

R version 4.3.2 (2023-10-31 ucrt)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19045)

Matrix products: default


locale:
[1] LC_COLLATE=English_VIR.utf8  LC_CTYPE=English_VIR.utf8    LC_MONETARY=English_VIR.utf8
[4] LC_NUMERIC=C                 LC_TIME=English_VIR.utf8    

time zone: America/New_York
tzcode source: internal

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] mlr3extralearners_0.8.0-9000 mlr3proba_0.6.0-9000         mlr3_0.16.1                 
[4] checkmate_2.3.1              testthat_3.2.1              

loaded via a namespace (and not attached):
 [1] remotes_2.4.2         stabs_0.6-4           rlang_1.1.3           magrittr_2.0.3       
 [5] compiler_4.3.2        vctrs_0.6.5           quadprog_1.5-8        stringr_1.5.1        
 [9] profvis_0.3.7         pkgconfig_2.0.3       crayon_1.5.2          fastmap_1.1.1        
[13] backports_1.4.1       ellipsis_0.3.2        inum_1.0-4            utf8_1.2.4           
[17] promises_1.2.1        sessioninfo_1.2.2     pracma_2.4.4          purrr_1.0.2          
[21] waldo_0.5.2           cachem_1.0.8          mlr3misc_0.13.0       jsonlite_1.8.8       
[25] collapse_1.8.9        later_1.3.2           uuid_1.2-0            parallel_4.3.2       
[29] R6_2.5.1              stringi_1.8.3         parallelly_1.36.0     pkgload_1.3.4        
[33] rpart_4.1.21          brio_1.1.4            xgboost_1.6.0.1       Rcpp_1.0.12          
[37] usethis_2.1.6         httpuv_1.6.14         Matrix_1.6-5          nnls_1.4             
[41] splines_4.3.2         tidyselect_1.2.0      rstudioapi_0.15.0     mlr3viz_0.6.1        
[45] partykit_1.2-16       codetools_0.2-19      miniUI_0.1.1.1        param6_0.2.4         
[49] listenv_0.9.0         pkgbuild_1.4.3        lattice_0.21-9        tibble_3.2.1         
[53] shiny_1.8.0           withr_3.0.0           future_1.33.1         desc_1.4.3           
[57] survival_3.5-7        urlchecker_1.0.1      pillar_1.9.0          generics_0.1.3       
[61] rprojroot_2.0.4       ggplot2_3.5.0         aorsf_0.1.3.9002      munsell_0.5.0        
[65] scales_1.3.0          dictionar6_0.1.3      globals_0.16.2        xtable_1.8-4         
[69] distr6_1.8.4          glue_1.7.0            tools_4.3.2           mlr3pipelines_0.5.0-1
[73] data.table_1.15.2     fs_1.6.3              mvtnorm_1.2-4         set6_0.2.6           
[77] grid_4.3.2            libcoin_1.0-9         devtools_2.4.4        colorspace_2.1-0     
[81] paradox_1.0.0         survivalmodels_0.1.13 palmerpenguins_0.1.0  Formula_1.2-5        
[85] cli_3.6.2             fansi_1.0.6           dplyr_1.1.4           ooplah_0.2.0         
[89] gtable_0.3.4          digest_0.6.34         lgr_0.4.3             htmlwidgets_1.6.4    
[93] memoise_2.0.1         htmltools_0.5.7       lifecycle_1.0.4       mboost_2.9-7         
[97] here_1.0.1            mime_0.12

Reply:
We can wait for 1) `paradox` and 2) `aorsf` CRAN updates before merging this commit, I don't see any problem with that, no hurries here.